docs/connect.html

```diff
@@ -183,6 +191,9 @@
 }
 </pre>
 
+Although not used in the example, <code>SourceTask</code> also provides two APIs to commit offsets in the source system: <code>commit</code> and <code>commitSourceRecord</code>. The APIs are provided for source systems that also need to store offsets internally which serve as an acknowledge mechanism to the source system that the message are delivered to Kafka.
+The <code>commit<code> API stores the offsets in the source system, up to the offsets that have been returned by <code>poll</code>. The implementation of this API should block until the commit is complete. The <code>commitSourceRecord</code> API saves the offset in the source system for each <code>SourceRecord</code> after it is delivered to Kafka.As Kafka Connect will record offsets automatically, <code>SourceTask<code>s are not required to implement them. Typically you only need to implement one of the two APIs.
```

Comment:
Missing space between sentences: Kafka.As -> Kafka. As

For the final sentence, since you just said most won't need to implement them, I'd start out with something like "In cases where a connector does need to acknowledge messages in the source system, only one of the APIs is typically required."